thetis/turbines.py

```diff
@@ -14,22 +14,31 @@
 class TidalTurbine:
     def __init__(self, options, upwind_correction=False):
         self.diameter = options.diameter
+        self.swept_diameter = options.swept_diameter or self.diameter
         self.C_support = options.C_support
         self.A_support = options.A_support
         self.upwind_correction = upwind_correction
 
+        # Check that the parameter choices make some sense - these won't break
+        # the simulation but may give unexpected results if the choice isn't
+        # understood.
+        if self.swept_diameter != self.diameter:
+            log(INFO, 'Warning - swept_diameter and plan_diameter are not equal. '
+                      'Upwind correction may not be accurate.')
+
```

Comment:
This might be a bit confusing: isn't the whole point of the swept_diameter option to be able to choose it differently from the diameter? Also there's no `plan_diameter` option